tools/pythonpkg/duckdb/__init__.py

```diff
@@ -3,11 +3,13 @@
 # Modules
 import duckdb.functional as functional
 import duckdb.typing as typing
+import duckdb.wal_reader as wal_reader
 import functools
 
 _exported_symbols.extend([
     "typing",
-    "functional"
+    "functional",
+    "wal_reader"
```

Comment:
@Tishj is this the right way to publish the bindings in `__init__.py`? This can only work if `duckdb/wal_reader/__init__.py` is installed correctly. My guess is that doing so requires changes to `setup.py`.

There seems to be some issue around `setup.py`. Perhaps I need to find the docs on how to make changes to it without breaking CI.

Reply:
I forgot to `git add duckdb/wal_reader/__init__.py`. Got mislead by other error messages in the error output. The most significant error was at the end and I was reading the error messages top down.